src/test/java/com/google/sps/MaxDepthTest.java

```diff
@@ -71,21 +70,19 @@ public void maxDepthZero() {
     nodeA.addChildren("B");
     nodeA.addChildren("C");
 
-    MutableGraph<GraphNode> graph = GraphBuilder.directed().build();
-
     HashMap<String, Node> protoNodesMap = new HashMap<>();
     protoNodesMap.put("A", nodeA.build());
     protoNodesMap.put("B", nodeB.build());
     protoNodesMap.put("C", nodeC.build());
 
-    HashMap<String, GraphNode> graphNodesMap = new HashMap<>();
-
-    HashSet<String> roots = new HashSet<>();
-
-    servlet.graphFromProtoNodes(protoNodesMap, graph, graphNodesMap, roots);
+    DataGraph dataGraph = new DataGraph();
+    dataGraph.graphFromProtoNodes(protoNodesMap);
+    MutableGraph<GraphNode> graph = dataGraph.getGraph();
+    HashMap<String, GraphNode> graphNodesMap = dataGraph.getGraphNodesMap();
+    HashSet<String> roots = dataGraph.getRoots();
 
     MutableGraph<GraphNode> truncatedGraph =
-        servlet.getGraphWithMaxDepth(graph, roots, graphNodesMap, 0);
+        Utility.getGraphWithMaxDepth(graph, roots, graphNodesMap, 0);
```

Comment:
usually it is better to import static method instead of import the entire class.

`import static com.google.sps.data.Utility.getGraphWithMaxDepth;`